spec/integration/fork/after/execution_output.txt

```diff
@@ -19,9 +19,10 @@ CLAIDE_DISABLE_AUTO_WRAP=TRUE COCOAPODS_SKIP_CACHE=TRUE COCOAPODS_VALIDATOR_SKIP
   local_sources: [],
   repo_update: false,
   use_default_plugins: false,
-  deterministic_uuids: false,
-  share_schemes_for_development_pods: true,
-  warn_for_multiple_pod_sources: false,
+  deterministic_uuids: true,
+  disable_input_output_paths: false,
+  share_schemes_for_development_pods: false,
```

Comment:
Is the default for `share_schemes_for_development_pods` effectively flipping from `true` to `false` an issue? Should this default instead to the pod being generated?

Reply:
The default value of [share_schemes_for_development_pods  in cocoapods](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/installation_options.rb#L165) is false. Do you still remember why we set it to true in pod gen? I can flip it back if that makes more sense.

Reply:
I think because `pod gen` aims to work with pods in a monorepo it was set to `true` but I do not recall 100%

Reply:
Actually I've already defaulted it to true if there is no podfile. The value in the test comes from the podfile. There are also tests whose share_schemes_for_development_pods is true. Should be safe.